promil/deploy/action.yaml

```diff
@@ -0,0 +1,63 @@
+name: 'Deploy promil'
+description: 'Deploy pointed branch of promil to pointed infra'
+inputs:
+  infrastructure_name:
+    required: True
+    description: Infrastructure name
+  github_token:
+    required: True
+    description: Github token with read and write access to Promil repo and infra repo.
+  ref:
+    required: False
+    description: Ref to commit in PiwikPRO/Promil repository
+    default: master
+  python-version:
+    required: false
+    description: Version of python
+    default: "3.10.x"
+  password:
+    required: false
+    description: Password to infrastructure
+  user:
+    required: false
+    description: This user name that will be author of commit in infra repository
```

Comment:
> This user name ~~that~~ will

I am not commenting on the articles (a, an, the)

Reply:
Better?